cfme/tests/storage/test_object_store_object.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+import pytest
+import random
+
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.storage.object_store_object import ObjectStoreObjectCollection
+from cfme.utils import testgen, version
+
+
+pytestmark = pytest.mark.usefixtures("setup_provider")
+
+
+pytest_generate_tests = testgen.generate([OpenStackProvider], scope="module")
+
+
+@pytest.yield_fixture(scope="module")
+def objects(appliance, provider):
+    collection = ObjectStoreObjectCollection(appliance=appliance)
+    objects = collection.all(provider)
+    yield objects
```

Comment:
**REQUIRED:** If it yields and exits, it may pretty much be a standard fixture since there is no finalizer.

Reply:
@mfalesni, you are right, I added this fixture in "model_collections.py" with returning  "objects" and "collection". Please correct me, if wrong.